src/Lykke.Service.PayAuth/Startup.cs

```diff
@@ -53,6 +54,9 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
                     options.DefaultLykkeConfiguration("v1", "PayAuth API");
                 });
 
+                Mapper.Initialize(x => x.AddProfiles(GetType().Assembly));
+                Mapper.Configuration.AssertConfigurationIsValid();
```

Comment:
Up to you, but maybe be it would be more logical to place this peace of code into unit test.